src/_includes/services_page_intro.liquid

```diff
@@ -1,5 +1,40 @@
 <header>
-  <div class="services-page-intro__image"></div>
+  <div class="services-page-intro__image">
+    <div class="services-page-intro__navigation">
+      <div class=services-page-intro__icon-container>
+        <a href="{{'/services/software-modernisation/' | prepend: site.baseurl }}" class="services-page-intro__navigation-link">
+          <div class="services-page-intro__navigation-icon services-page-intro__icon--software-modernisation">
```

Comment:
If we were to follow the BEM naming convention,  modifier class would replicate the original with the modifier appended to end. 

i.e. `services-page-intro__navigation-icon--software-modernisation`